bedrock/settings/__init__.py

```diff
@@ -123,6 +143,45 @@
         FLUENT_REPO_PATH,
     ]
 
+    CANONICAL_LOCALES = {
+        # TODO: check whether redundant when we have real translations flowing in to Bedrock
+        "en-US": "en",
+        "en-GB": "en",
+        "en-CA": "en",
+        "es-ES": "es-ES",
+    }
```

Comment:
Some of these may also be unnecessary since the middleware will match on the 2 letter locale by default. The reverse is not true which is why bedrock needs, e.g. `en` -> `en-US`.